app/models/friendship.rb

```diff
@@ -4,4 +4,10 @@ class Friendship < ApplicationRecord
   belongs_to :user
   belongs_to :friend, class_name: 'User'
   validates :user, uniqueness: { scope: :friend }
+ 
+  def confirm_friend(user)
+    friendship = inverse_friendships.find { |f| f.user == user and f.friend == self }
+    friendship.confirmed = true
+    friendship.save
```

Comment:
- [ ] When a friendship is accepted, a new row with the inverse relation of the friendship should be created.

Reply:
Perfect, thanks